src/main/java/com/esri/samples/displayinformation/add_graphics_with_renderer/README.md

```diff
@@ -1,28 +1,20 @@
 <h1>Add Graphics with Renderer</h1>
 
-<p>Demonstrates how to add Graphics to a GraphicsOverlay and display those Graphics to a MapView using a UniqueValueRenderer. The UniqueValueRenderer uses the attribute value of a Graphic to assign a specific symbol to a specific graphic. </p>
+<p>Demonstrates how to add graphics to a graphics overlay and display them with a renderer. A renderer allows you to 
```

Comment:
Should probably state that a renderer only changes the symbol for graphics that don't have a symbol assigned to them when created.